pkgs/applications/virtualization/podman/wrapper.nix

```diff
@@ -0,0 +1,47 @@
+{ podman-unwrapped
+, runCommandNoCC
+, makeWrapper
+, lib
+, extraPackages ? []
+, podman # Docker compat
+, runc # Default container runtime
+, crun # Default container runtime (cgroups v2)
+, conmon # Container runtime monitor
+, slirp4netns # User-mode networking for unprivileged namespaces
+, fuse-overlayfs # CoW for images, much faster than default vfs
+, utillinux # nsenter
+, iptables
```

Comment:
What is the reason for not adding cni-plugins here?

Reply:
Added.